src/cloudimagedirectory/transform/transform.py

```diff
@@ -41,6 +42,62 @@ def run(self, data):
         raise NotImplementedError
 
 
+class TransformerIdxListImageLatest(Transformer):
+    """Transform raw AWS data."""
+
+    def run(self, data):
+        """Transform the raw data."""
+        # Verify that the data is from AWS.
+        entries = [x for x in data if not x.is_raw()]
+
+        # Sort the list of data by date
+        entries.sort(
+            key=lambda x: datetime.strptime(
+                "".join(x.content["date"].split("T")[0]), "%Y-%m-%d"
+            ),
+            reverse=True,
+        )
+
+        list = []
+        for e in entries:
```

Comment:
In Python, single letter variables are avoided outside of small structures like lambdas or dictionary comprehension.

`for entry in entries` would be fine here.

Reply:
aaah, right - will change it :smile: